src/parser/github-comment-module.ts

```diff
@@ -52,6 +52,9 @@ export class GithubCommentModule implements Module {
         result[key].evaluationCommentHtml = await this._generateHtml(key, value, true);
         strippedBody.push(result[key].evaluationCommentHtml);
       }
+      strippedBody.push("\n<!--");
+      strippedBody.push(this._encodeHTML(`\n${getGithubWorkflowRunUrl()}`));
```

Comment:
Where are the standardized headers? Body should be json compliant. 

 This needs to be used across all plugins for a single parser to work

Reply:
Where are these headers documented? It was never implemented in this plugin afaik

Reply:
Added metadata in https://github.com/ubiquibot/conversation-rewards/pull/121/commits/bd32e7393ae4181aafb86e48e6c4b753787a69de